pcsx2/gui/FrameForGS.cpp

```diff
@@ -623,6 +637,26 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 	out << std::fixed << std::setprecision(2) << fps;
 	OSDmonitor(Color_StrongGreen, "FPS:", out.str());
 
+	int new_width = 0, new_height = 0;
+	AspectRatioType Aspect_Ratio = g_Conf->GSWindow.AspectRatio;
+	if (GSgetImageSize != nullptr && (Aspect_Ratio == AspectRatio_Frame || (Aspect_Ratio != AspectRatio_Stretch &&
+																			g_Conf->GSWindow.ScalingType != ScalingType_Fit))) {
+		GSgetImageSize(&new_width, &new_height);
+		if (new_width > 0 && new_height > 0)
+		{
+			if (GSPanel* gsPanel = GetViewport())
+			{
+				if (image_width != new_width || image_height != new_height)
```

Comment:
This commit doesn't compile. Should be `s_image_width`, `s_image_height`.

Reply:
Fixed.